app/scripts/controllers/courtSearchInfo.ctrl.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+angular.module('yourStlCourts').controller('courtSearchInfoCtrl', function ($state, $window, courtInfo) {
+  var ctrl = this;
+  ctrl.courtOnMap = {
+        lat:51.505,
+        lng:-0.09,
+        zoom: 14
+  };
+  ctrl.courtMarkers = {};
+  var courtDefaultIcon = {
+    iconUrl: 'images/marker-icon.png',
+    shadowUrl: 'images/marker-shadow.png',
+    iconSize: [25, 41],
+    iconAnchor: [12, 41],
+    popupAnchor: [1, -34],
+    shadowSize: [41, 41]
+  };
+
+
+
+
+
+
+  if(!courtInfo) {
+    $state.go('home');
+  } else {
+    ctrl.courtInfo = courtInfo;
+    ctrl.courtDirectionLink = getCourtDirectionLink(courtInfo);
+    ctrl.courtOnMap.lat = courtInfo.latitude;
+    ctrl.courtOnMap.lng = courtInfo.longitude;
+    ctrl.courtMarkers.m1 = {lat:courtInfo.latitude,lng:courtInfo.longitude, message: "Test",icon: courtDefaultIcon};
+    //ctrl.courtMarker.lat = courtInfo.latitude;
```

Comment:
It's rare that leaving commented out code adds value. Make sure you remove these in the future